Samples/ResourceManagement/cs-winforms-unpackaged/winforms_unpackaged_app.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ProjectReunion" Version="0.8.0-preview" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0-preview1" />
```

Comment:
In case you didn't know: verifying unpackaged apps requires some additional steps. Please see https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/deploy-unpackaged-apps#prerequisites.